Prox/Prox/PlaceDetails/PlaceDetailsCardView.swift

```diff
@@ -327,7 +327,7 @@ class PlaceDetailsCardView: UIView {
 
         hoursView.iconView.isHidden = false
         hoursView.isPrimaryTextLabelHidden = false
-        hoursView.primaryTextLabel.text = primaryText
+        hoursView.primaryTextLabel.text = primaryText.capitalized
```

Comment:
I'm concerned this means we'll be capitalizing something we didn't mean to (e.g. error state)... But whatever.